atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1940,23 +1940,24 @@ void WebContents::SetZoomFactor(double factor) {
 }
 
 double WebContents::GetZoomFactor() const {
-  auto level = GetZoomLevel();
+  double level = zoom_controller_->GetZoomLevel();
   return content::ZoomLevelToZoomFactor(level);
 }
 
 void WebContents::OnSetTemporaryZoomLevel(content::RenderFrameHost* rfh,
                                           double level,
                                           IPC::Message* reply_msg) {
   zoom_controller_->SetTemporaryZoomLevel(level);
-  double new_level = zoom_controller_->GetZoomLevel();
+  double new_level = GetZoomFactor();
```

Comment:
Shouldn't be changed from zoom level to zoom factor.

Reply:
oops, copy/paste mistake!